front/src/core-logic/domain/admin/icUsersAdmin/icUsersAdmin.epics.ts

```diff
@@ -116,6 +117,33 @@ const rejectAgencyToUserEpic: AppEpic<IcUsersAdminAction> = (
     ),
   );
 
+const updateUserOnAgencyEpic: AppEpic<IcUsersAdminAction> = (
+  action$,
+  state$,
+  { adminGateway },
+) =>
+  action$.pipe(
+    filter(icUsersAdminSlice.actions.updateUserOnAgencyRequested.match),
+    switchMap(
+      (action: PayloadActionWithFeedbackTopic<IcUserRoleForAgencyParams>) =>
```

Comment:
ce typage n'est pas necessaire , normalement tu en hérite directement grace au filter au dessus:

```suggestion
      (action) =>
```

Reply:
pris en compte